client/js/store/procedure/SegmentFilter.js

```diff
@@ -11,7 +11,9 @@ import { del, set } from 'vue'
 
 const SegmentFilterStore = {
   namespaced: true,
-  name: 'segmentfilter',
+
+  name: 'Segmentfilter',
```

Comment:
Why not `SegmentFilter`?

Reply:
because I just batch-renamed...
Done here cd3496ceffc24c9475b3de7f2a0bb9bf76cd35c8